src/app/nextversion/nextversion_test.go

```diff
@@ -346,7 +383,7 @@ dependencies:
 		tc := tc
 		//nolint:paralleltest // urfave/cli cannot be tested concurrently.
 		t.Run(tc.name, func(t *testing.T) {
-			repoDir := repoWithTags(t, tc.tags...)
+			repoDir := t.TempDir()
```

Comment:
I expected the repository to be initialized with something like

```go
repoDir := repoWithTags(t) // repo created with no tags
```

But now I reckon it to be a different test case.  As I see it, a repository with no tags and a directory (which is not a git repository) are different failure test cases. 

If we wanted to cover both, we could define the repository initialization as one of the test arguments so we could use:

```go
repoDir := tc.repo(t)
```

WDYT?

Reply:
I think it's a good point, I should have used a repo with no tags. I'm not sure why I went for the empty folder 🤔 

From what the command is concerned, there shouldn't be any difference though. But... it's easy enough to just provide the function :P

Reply:
Done in https://github.com/newrelic/release-toolkit/pull/133/commits/8527223216e8d02eb0c2a17faa5d634e8be0cac3